faq.md

```diff
@@ -0,0 +1,101 @@
+# Frequently Asked Questions (FAQ)
+
+## How do I install clangd?
+
+Clangd is often distributed either within LLVM packages or in a separate
+Clang-related packages (e.g.
+[clang-tools](https://packages.ubuntu.com/search?keywords=clang-tools) on
+Ubuntu) but these packages are typically outdated.
+
+If you want to use new versions of clangd, you have several options:
+
+- Download the binaries from our GitHub repository:
+  - [Stable track](https://github.com/clangd/clangd/releases/latest)
+  - [Unstable weekly builds](https://github.com/clangd/clangd/releases)
```

Comment:
I'd rather say these are weekly snapshots and not tested as thoroughly as official releases.